pages/linux/xxd.md

```diff
@@ -0,0 +1,24 @@
+# xxd
+
+> Make a hexdump or do the reverse.
+> More information: <https://packages.debian.org/buster/xxd>.
+
+- Display Hex representation of a file:
+
+`xxd {{file}}`
```

Comment:
Does this expect only the filename or a path?

Reply:
A path to the file to be specific.
If the file is in current working dir then just file name is enough.